docs/parquet.md

```diff
@@ -48,3 +48,14 @@ implicit val pfDecimalBinary = ParquetField.decimalBinary(20, 0)
 Among the date/time types, `DATE` maps to `java.time.LocalDate`. The other types, `TIME` and `TIMESTAMP`, map to `OffsetTime`/`LocalTime` and `Instant`/`LocalDateTime` with `isAdjustedToUTC` set to `true`/`false`. They can be in nano, micro, or milliseconds precision with `import magnolify.parquet.logical.{nanos,micros,millis}._`.
 
 Note that Parquet's official Avro support maps `REPEATED` fields to an `array` field inside a nested group. Use `import magnolify.parquet.ParquetArray.AvroCompat._` to ensure compatibility with Avro.
+
+The top level class and all fields (including nested class fields) can be annotated with `@doc` annotation. Note that nested classes annotations are ignored.
```

Comment:
The reason: existing Spotify avro annotations do not support type-level annotations, only field-level:
https://backstage.spotify.net/docs/default/component/data-protection-privacy/access/schemas/avsc/#my-type-doc-annotation-isnt-being-parsed-properly

Exception: the top-level type, which annotation is describing the entire dataset.
For me the intuition is simple: Spotify's data policies are concerned about data fields, because they store actual data - either sensitive or not. But nested types are auxiliary & artificial entity. According to the link above we may skip those annotations in parquet as well

Reply:
This is internal to Spotify.

As OSS library, I would consider type-level annotation as a fallback if field level annotation isn't defined.
Doing so, user can define the doc once on the nested class instead of repeating the doc on every field where the nested class is used.

Reply:
@RustedBones we can do type-level annotation fallback, but  I am afraid it contradicts to the existing Avro approach, so it may confuse our Spotify developers, especially if some teams support two data types